spec/features/event_disavowal_spec.rb

```diff
@@ -22,14 +22,35 @@
     disavow_last_action_and_reset_password
   end
 
-  scenario 'disavowing a new device sign in' do
-    allow(IdentityConfig.store).to receive(:otp_delivery_blocklist_maxretry).and_return(3)
-    signin(user.email, user.password)
-    Capybara.reset_session!
-    visit root_path
-    signin(user.email, user.password)
+  context 'when aggregated new device alerts is disabled' do
+    before do
+      allow(IdentityConfig.store).to receive(
+        :feature_new_device_alert_aggregation_enabled,
+      ).and_return(false)
+    end
+    scenario 'disavowing a new device sign in' do
+      allow(IdentityConfig.store).to receive(:otp_delivery_blocklist_maxretry).and_return(3)
+      signin(user.email, user.password)
+      Capybara.reset_session!
+      visit root_path
+      signin(user.email, user.password)
+      disavow_last_action_and_reset_password
+    end
+  end
 
-    disavow_last_action_and_reset_password
+  context 'when aggregated new device alerts is enabled' do
+    before do
+      allow(IdentityConfig.store).to receive(
+        :feature_new_device_alert_aggregation_enabled,
+      ).and_return(true)
+    end
+    scenario 'disavowing a new device sign in' do
+      allow(IdentityConfig.store).to receive(:otp_delivery_blocklist_maxretry).and_return(3)
+      signin(user.email, user.password)
+      Capybara.reset_session!
+      visit root_path
+      signin(user.email, user.password)
```

Comment:
What are we asserting in this test case?

Reply:
It seems like it's not needed given the output of the opposite case ultimately checks that the disavowable email is sent and in this iteration the email doesn't get sent. I'm not sure really what useful test can go there for the present. Should I remove it? Once it's wired up it'll be useful to make sure the disavowal continues to go through...

Reply:
Yeah I think we can remove for now and add that test as part of wiring it up.